torch/csrc/jit/passes/onnx/cast_constant.cpp

```diff
@@ -0,0 +1,71 @@
+#include <torch/csrc/jit/passes/onnx/cast_constant.h>
+#include <onnx/onnx_pb.h>
```

Comment:
why do we need onnx header here?

Reply:
in order to use `::ONNX_NAMESPACE::TensorProto_DataType_INT64` instead of hard coding type value.

Reply:
only two uses of ONNX data type, it's fine to hard code them with some comments. I would like to let  clang tidy to check this file :-)